forest/tree/tree.go

```diff
@@ -0,0 +1,58 @@
+package tree
+
+import (
+	cache "github.com/treeverse/lakefs/forest/cache_map"
+	gr "github.com/treeverse/lakefs/graveler"
+	"github.com/treeverse/lakefs/graveler/committed/sstable"
+)
+
+var minimalKey = gr.Key(nil)
+
+type treePartType struct {
+	PartName sstable.ID `json:"part_name"`
+	MaxKey   gr.Key     `json:"max_path"`
+}
+type TreeType struct {
```

Comment:
please remove the word "type", it's confusing.
Also from the treePartType

Reply:
Done